Rakefile

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+require 'date'
+require 'json'
+require 'net/http'
+require 'optparse'
+
+require 'curb' # Curl Ruby, an external libcurl binding library
+
+UNDESIRED_LABELS = [
+  'refactoring', 'needs conceptual review', 'needs rebase', 'waiting for author'
+]
+GITHUB_API_URL = 'https://api.github.com/repos/bitcoin/bitcoin/pulls'
+HTTP_SUCCESS = 200
+HTTP_ERRORS = [
+  JSON::ParserError,
+  EOFError,
+  Errno::ECONNRESET,
+  Errno::EINVAL,
+  Net::HTTPBadResponse,
+  Net::HTTPHeaderSyntaxError,
+  Net::ProtocolError,
+  Timeout::Error
+]
+
+desc 'Create a new post file'
+namespace :posts do
+  task :new do
+    # Fetch user command line arguments.
+    pr, host, date = get_cli_options
+
+    # Set default value for meeting date if not supplied by user.
+    date ||= next_wednesday.to_s
```

Comment:
It'd be nice if this printed out a message to the user like `date set to next Wednesday: <date>`

Reply:
Good idea. Done.